addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/handler/FreeboxHandler.java

```diff
@@ -118,9 +126,14 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     break;
                 case REBOOT:
                     setReboot(command);
+                default:
+                    logger.debug("Thing {}: unexpected command {} from channel {}", getThing().getUID(), command,
+                            channelUID.getId());
+                    break;
             }
-        } catch (FreeboxException e) {
-            logger.error("{}", e.getMessage());
+        } catch (Exception e) {
```

Comment:
Imho, if there is no way you can really handle (ie recover from it) it you should pass it one to the caller.

The framework does handle it in: https://github.com/eclipse/smarthome/blob/648e22c80a49277cbba30c1d0d08e1444073bd04/bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java#L373